docs/userguide/async.md

```diff
@@ -0,0 +1,74 @@
+# Asynchronous Workflows
+
+Asynchronous workflows describe situations where the loop between measurements and
+recommendations is more complex and needs to incorporate various other aspects. These
+could for instance be:
+- **Distributed workflows**: When recommendations are distributed across several
+  operators, e.g. at different locations or in several reactors, some experiments might
```

Comment:
perhaps a generalize using a different word than "reactors"? Pieces of equipment? Measurement devices? Lab equipment? Maybe you have a good idea...

Reply:
this is a concrete example started with `e.g.`, no need to provide a general statement after that